diff --git a/node_modules/@graphql-tools/load/esm/documents.js b/node_modules/@graphql-tools/load/esm/documents.js
index a229882..6dba2da 100644
--- a/node_modules/@graphql-tools/load/esm/documents.js
+++ b/node_modules/@graphql-tools/load/esm/documents.js
@@ -1,4 +1,4 @@
-import { Kind } from 'graphql';
+import { Kind } from 'graphql/index.mjs';
 import { loadTypedefs, loadTypedefsSync } from './load-typedefs.js';
 /**
  * Kinds of AST nodes that are included in executable documents
diff --git a/node_modules/@graphql-tools/load/esm/filter-document-kind.js b/node_modules/@graphql-tools/load/esm/filter-document-kind.js
index 75acad2..5948863 100644
--- a/node_modules/@graphql-tools/load/esm/filter-document-kind.js
+++ b/node_modules/@graphql-tools/load/esm/filter-document-kind.js
@@ -1,4 +1,4 @@
-import { Kind } from 'graphql';
+import { Kind } from 'graphql/index.mjs';
 import { env } from 'process';
 /**
  * @internal
diff --git a/node_modules/@graphql-tools/load/esm/load-typedefs/collect-sources.js b/node_modules/@graphql-tools/load/esm/load-typedefs/collect-sources.js
index fadbab7..83d8bc7 100644
--- a/node_modules/@graphql-tools/load/esm/load-typedefs/collect-sources.js
+++ b/node_modules/@graphql-tools/load/esm/load-typedefs/collect-sources.js
@@ -1,5 +1,5 @@
 import { isDocumentString, parseGraphQLSDL, getDocumentNodeFromSchema, asArray } from '@graphql-tools/utils';
-import { isSchema, Kind } from 'graphql';
+import { isSchema, Kind } from 'graphql/index.mjs';
 import { loadFile, loadFileSync } from './load-file.js';
 import { stringToHash, useStack } from '../utils/helpers.js';
 import { useCustomLoader, useCustomLoaderSync } from '../utils/custom-loader.js';
diff --git a/node_modules/@graphql-tools/load/esm/schema.js b/node_modules/@graphql-tools/load/esm/schema.js
index 4a0eda2..dce392e 100644
--- a/node_modules/@graphql-tools/load/esm/schema.js
+++ b/node_modules/@graphql-tools/load/esm/schema.js
@@ -1,5 +1,5 @@
 import { loadTypedefs, loadTypedefsSync } from './load-typedefs.js';
-import { Source as GraphQLSource, print, lexicographicSortSchema } from 'graphql';
+import { Source as GraphQLSource, print, lexicographicSortSchema } from 'graphql/index.mjs';
 import { OPERATION_KINDS } from './documents.js';
 import { mergeSchemas, extractExtensionsFromSchema } from '@graphql-tools/schema';
 import { getResolversFromSchema } from '@graphql-tools/utils';
