diff --git a/node_modules/@graphql-tools/git-loader/esm/index.js b/node_modules/@graphql-tools/git-loader/esm/index.js
index 59d7cbe..d844872 100644
--- a/node_modules/@graphql-tools/git-loader/esm/index.js
+++ b/node_modules/@graphql-tools/git-loader/esm/index.js
@@ -3,7 +3,7 @@ import micromatch from 'micromatch';
 import unixify from 'unixify';
 import { loadFromGit, loadFromGitSync, readTreeAtRef, readTreeAtRefSync } from './load-git.js';
 import { parse as handleStuff } from './parse.js';
-import { parse } from 'graphql';
+import { parse } from 'graphql/index.mjs';
 import { asArray } from '@graphql-tools/utils';
 import isGlob from 'is-glob';
 import { env } from 'process';
