diff --git a/node_modules/@graphql-tools/import/esm/index.js b/node_modules/@graphql-tools/import/esm/index.js
index 6152ccb..259ac72 100644
--- a/node_modules/@graphql-tools/import/esm/index.js
+++ b/node_modules/@graphql-tools/import/esm/index.js
@@ -1,4 +1,4 @@
-import { Source, parse, Kind, print, } from 'graphql';
+import { Source, parse, Kind, print, } from 'graphql/index.mjs';
 import { readFileSync, realpathSync } from 'fs';
 import { dirname, join, isAbsolute } from 'path';
 import resolveFrom from 'resolve-from';
