diff --git a/node_modules/@graphql-tools/code-file-loader/esm/exports.js b/node_modules/@graphql-tools/code-file-loader/esm/exports.js
index 10b3144..4b2ff69 100644
--- a/node_modules/@graphql-tools/code-file-loader/esm/exports.js
+++ b/node_modules/@graphql-tools/code-file-loader/esm/exports.js
@@ -1,4 +1,4 @@
-import { parse, buildClientSchema, isSchema } from 'graphql';
+import { parse, buildClientSchema, isSchema } from 'graphql/index.mjs';
 import { isSchemaAst, isSchemaJson, isSchemaText, isWrappedSchemaJson, pick } from './helpers.js';
 const identifiersToLookFor = ['default', 'schema', 'typeDefs', 'data'];
 // Pick exports
diff --git a/node_modules/@graphql-tools/code-file-loader/esm/index.js b/node_modules/@graphql-tools/code-file-loader/esm/index.js
index ea13108..57d9427 100644
--- a/node_modules/@graphql-tools/code-file-loader/esm/index.js
+++ b/node_modules/@graphql-tools/code-file-loader/esm/index.js
@@ -1,4 +1,4 @@
-import { isSchema, parse } from 'graphql';
+import { isSchema, parse } from 'graphql/index.mjs';
 import { asArray, isValidPath, parseGraphQLSDL, isDocumentNode, } from '@graphql-tools/utils';
 import { gqlPluckFromCodeString, gqlPluckFromCodeStringSync, } from '@graphql-tools/graphql-tag-pluck';
 import globby from 'globby';
