diff --git a/node_modules/@graphql-tools/url-loader/esm/index.js b/node_modules/@graphql-tools/url-loader/esm/index.js
index 853559e..d201d02 100644
--- a/node_modules/@graphql-tools/url-loader/esm/index.js
+++ b/node_modules/@graphql-tools/url-loader/esm/index.js
@@ -1,4 +1,4 @@
-import { buildASTSchema, buildSchema } from 'graphql';
+import { buildASTSchema, buildSchema } from 'graphql/index.mjs';
 import { parseGraphQLSDL, getOperationASTFromRequest, } from '@graphql-tools/utils';
 import { schemaFromExecutor, wrapSchema } from '@graphql-tools/wrap';
 import WebSocket from 'isomorphic-ws';
