diff --git a/node_modules/@graphql-tools/executor/esm/execution/execute.js b/node_modules/@graphql-tools/executor/esm/execution/execute.js
index 3d5442a..802029f 100644
--- a/node_modules/@graphql-tools/executor/esm/execution/execute.js
+++ b/node_modules/@graphql-tools/executor/esm/execution/execute.js
@@ -1,4 +1,4 @@
-import { locatedError, Kind, isAbstractType, isLeafType, isListType, isNonNullType, isObjectType, assertValidSchema, getDirectiveValues, SchemaMetaFieldDef, TypeMetaFieldDef, TypeNameMetaFieldDef, GraphQLError, } from 'graphql';
+import { locatedError, Kind, isAbstractType, isLeafType, isListType, isNonNullType, isObjectType, assertValidSchema, getDirectiveValues, SchemaMetaFieldDef, TypeMetaFieldDef, TypeNameMetaFieldDef, GraphQLError, } from 'graphql/index.mjs';
 import { createGraphQLError, inspect, isAsyncIterable, isIterableObject, isObjectLike, isPromise, pathToArray, addPath, getArgumentValues, promiseReduce, memoize3, getDefinedRootType, mapAsyncIterator, GraphQLStreamDirective, collectFields, collectSubFields as _collectSubfields, memoize1, } from '@graphql-tools/utils';
 import { getVariableValues } from './values.js';
 import { promiseForObject } from './promiseForObject.js';
diff --git a/node_modules/@graphql-tools/executor/esm/execution/normalizedExecutor.js b/node_modules/@graphql-tools/executor/esm/execution/normalizedExecutor.js
index 0558b6d..906e75d 100644
--- a/node_modules/@graphql-tools/executor/esm/execution/normalizedExecutor.js
+++ b/node_modules/@graphql-tools/executor/esm/execution/normalizedExecutor.js
@@ -1,4 +1,4 @@
-import { getOperationAST } from 'graphql';
+import { getOperationAST } from 'graphql/index.mjs';
 import { execute, flattenIncrementalResults, subscribe } from './execute.js';
 import { ValueOrPromise } from 'value-or-promise';
 export function normalizedExecutor(args) {
diff --git a/node_modules/@graphql-tools/executor/esm/execution/values.js b/node_modules/@graphql-tools/executor/esm/execution/values.js
index 3cfb0e4..4a11e7a 100644
--- a/node_modules/@graphql-tools/executor/esm/execution/values.js
+++ b/node_modules/@graphql-tools/executor/esm/execution/values.js
@@ -1,4 +1,4 @@
-import { print, isInputType, isNonNullType, coerceInputValue, typeFromAST, valueFromAST, } from 'graphql';
+import { print, isInputType, isNonNullType, coerceInputValue, typeFromAST, valueFromAST, } from 'graphql/index.mjs';
 import { createGraphQLError, hasOwnProperty, inspect, printPathArray } from '@graphql-tools/utils';
 /**
  * Prepares an object map of variableValues of the correct type based on the
