diff --git a/node_modules/@graphql-tools/mock/esm/MockStore.js b/node_modules/@graphql-tools/mock/esm/MockStore.js
index 6a6f61f..b2b009b 100644
--- a/node_modules/@graphql-tools/mock/esm/MockStore.js
+++ b/node_modules/@graphql-tools/mock/esm/MockStore.js
@@ -1,4 +1,4 @@
-import { GraphQLString, isObjectType, isScalarType, getNullableType, isListType, isEnumType, isAbstractType, isCompositeType, isNullableType, isInterfaceType, } from 'graphql';
+import { GraphQLString, isObjectType, isScalarType, getNullableType, isListType, isEnumType, isAbstractType, isCompositeType, isNullableType, isInterfaceType, } from 'graphql/index.mjs';
 import stringify from 'fast-json-stable-stringify';
 import { isRef, assertIsRef, isRecord, } from './types.js';
 import { uuidv4, randomListLength, takeRandom, makeRef } from './utils.js';
diff --git a/node_modules/@graphql-tools/mock/esm/addMocksToSchema.js b/node_modules/@graphql-tools/mock/esm/addMocksToSchema.js
index 7eaa359..7b4f9a8 100644
--- a/node_modules/@graphql-tools/mock/esm/addMocksToSchema.js
+++ b/node_modules/@graphql-tools/mock/esm/addMocksToSchema.js
@@ -1,4 +1,4 @@
-import { defaultFieldResolver, isUnionType, GraphQLUnionType, GraphQLInterfaceType, isSchema, } from 'graphql';
+import { defaultFieldResolver, isUnionType, GraphQLUnionType, GraphQLInterfaceType, isSchema, } from 'graphql/index.mjs';
 import { mapSchema, MapperKind } from '@graphql-tools/utils';
 import { addResolversToSchema } from '@graphql-tools/schema';
 import { isRef } from './types.js';
diff --git a/node_modules/@graphql-tools/mock/esm/mockServer.js b/node_modules/@graphql-tools/mock/esm/mockServer.js
index 9ef2d0c..d1c3a3b 100644
--- a/node_modules/@graphql-tools/mock/esm/mockServer.js
+++ b/node_modules/@graphql-tools/mock/esm/mockServer.js
@@ -1,4 +1,4 @@
-import { isSchema, graphql } from 'graphql';
+import { isSchema, graphql } from 'graphql/index.mjs';
 import { makeExecutableSchema } from '@graphql-tools/schema';
 import { addMocksToSchema } from './addMocksToSchema.js';
 /**
