diff --git a/node_modules/@graphql-tools/graphql-tag-pluck/esm/index.js b/node_modules/@graphql-tools/graphql-tag-pluck/esm/index.js
index 28e2593..766b813 100644
--- a/node_modules/@graphql-tools/graphql-tag-pluck/esm/index.js
+++ b/node_modules/@graphql-tools/graphql-tag-pluck/esm/index.js
@@ -4,7 +4,7 @@ import { getExtNameFromFilePath } from './libs/extname.js';
 import createVisitor from './visitor.js';
 import traversePkg from '@babel/traverse';
 import { freeText } from './utils.js';
-import { Source } from 'graphql';
+import { Source } from 'graphql/index.mjs';
 function getDefault(module) {
     return module.default || module;
 }
