diff --git a/node_modules/@graphql-tools/batch-execute/esm/mergeRequests.js b/node_modules/@graphql-tools/batch-execute/esm/mergeRequests.js
index babc744..a3944c3 100644
--- a/node_modules/@graphql-tools/batch-execute/esm/mergeRequests.js
+++ b/node_modules/@graphql-tools/batch-execute/esm/mergeRequests.js
@@ -1,5 +1,5 @@
 // adapted from https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-graphql/src/batching/merge-queries.js
-import { visit, Kind, } from 'graphql';
+import { visit, Kind, } from 'graphql/index.mjs';
 import { getOperationASTFromRequest } from '@graphql-tools/utils';
 import { createPrefix } from './prefix.js';
 /**
