diff --git a/node_modules/@graphql-tools/relay-operation-optimizer/esm/index.js b/node_modules/@graphql-tools/relay-operation-optimizer/esm/index.js
index 03e60c7..e1f21c0 100644
--- a/node_modules/@graphql-tools/relay-operation-optimizer/esm/index.js
+++ b/node_modules/@graphql-tools/relay-operation-optimizer/esm/index.js
@@ -1,5 +1,5 @@
 import { printSchemaWithDirectives } from '@graphql-tools/utils';
-import { parse, concatAST } from 'graphql';
+import { parse, concatAST } from 'graphql/index.mjs';
 import { transform as skipRedundantNodesTransform } from '@ardatan/relay-compiler/lib/transforms/SkipRedundantNodesTransform.js';
 import { transform as inlineFragmentsTransform } from '@ardatan/relay-compiler/lib/transforms/InlineFragmentsTransform.js';
 import { transform as applyFragmentArgumentTransform } from '@ardatan/relay-compiler/lib/transforms/ApplyFragmentArgumentTransform.js';
