diff --git a/node_modules/@graphql-tools/optimize/esm/optimizers/remove-description.js b/node_modules/@graphql-tools/optimize/esm/optimizers/remove-description.js
index 203c39a..3db7d7d 100644
--- a/node_modules/@graphql-tools/optimize/esm/optimizers/remove-description.js
+++ b/node_modules/@graphql-tools/optimize/esm/optimizers/remove-description.js
@@ -1,4 +1,4 @@
-import { visit } from 'graphql';
+import { visit } from 'graphql/index.mjs';
 /**
  * This optimizer removes "description" field from schema AST definitions.
  * @param input
diff --git a/node_modules/@graphql-tools/optimize/esm/optimizers/remove-empty-nodes.js b/node_modules/@graphql-tools/optimize/esm/optimizers/remove-empty-nodes.js
index ce3b0d0..f5b347e 100644
--- a/node_modules/@graphql-tools/optimize/esm/optimizers/remove-empty-nodes.js
+++ b/node_modules/@graphql-tools/optimize/esm/optimizers/remove-empty-nodes.js
@@ -1,4 +1,4 @@
-import { visit } from 'graphql';
+import { visit } from 'graphql/index.mjs';
 /**
  * This optimizer removes empty nodes/arrays (directives/argument/variableDefinitions) from a given DocumentNode of operation/fragment.
  * @param input
diff --git a/node_modules/@graphql-tools/optimize/esm/optimizers/remove-loc.js b/node_modules/@graphql-tools/optimize/esm/optimizers/remove-loc.js
index a7d105b..afcde1e 100644
--- a/node_modules/@graphql-tools/optimize/esm/optimizers/remove-loc.js
+++ b/node_modules/@graphql-tools/optimize/esm/optimizers/remove-loc.js
@@ -1,4 +1,4 @@
-import { visit } from 'graphql';
+import { visit } from 'graphql/index.mjs';
 /**
  * This optimizer removes "loc" fields
  * @param input
