diff --git a/node_modules/@graphql-tools/executor-graphql-ws/esm/index.js b/node_modules/@graphql-tools/executor-graphql-ws/esm/index.js
index c7bf9f4..90d1431 100644
--- a/node_modules/@graphql-tools/executor-graphql-ws/esm/index.js
+++ b/node_modules/@graphql-tools/executor-graphql-ws/esm/index.js
@@ -1,5 +1,5 @@
 import { getOperationASTFromRequest } from '@graphql-tools/utils';
-import { print } from 'graphql';
+import { print } from 'graphql/index.mjs';
 import { createClient } from 'graphql-ws';
 import WebSocket from 'isomorphic-ws';
 function isClient(client) {
